Backend/ReSharperPlugin/ForTea.nuspec

```diff
@@ -2,18 +2,18 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
 	<metadata>
 		<id>JetBrains.ForTea</id>
-		<version>2023.1.0</version>
+		<version>$pluginVersion$</version>
 		<title>ForTea</title>
-		<authors>Kirill Glazyrin, JetBrains; Julien Lebosquain</authors>
+		<authors>Kirill Glazyrin; Julien Lebosquain</authors>
```

Comment:
My understanding is that JetBrains is still supposed to be listed here.

Or am I incorrect?